Tests/AnnotatedMock.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace SuperBrave\GdprBundle\Tests;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use SuperBrave\GdprBundle\Annotation as GDPR;
+
+/**
+ * AnnotatedMock.
+ *
+ * @author Niels Nijens <nn@superbrave.nl>
+ */
+class AnnotatedMock
+{
+    /**
+     * @GDPR\Export()
+     *
+     * @var string
+     */
+    private $foo = 'bar';
+
+    /**
+     * @GDPR\Export()
+     *
+     * @var int
+     */
+    private $baz = 1;
+
+    /**
+     * @GDPR\Export()
+     *
+     * @var array
+     */
+    private $qux = array();
+
+    /**
+     * @GDPR\Export()
+     *
+     * @var ArrayCollection
+     */
+    private $quux;
+
```

Comment:
Can you add a test for fields that aren't annotated?

Reply:
Added the `notAnnotatedProperty` property. 😁